### PR TITLE
adds a null check to CanReach()

### LINKED ISSUE #68365: Clicking on the fadein after spawning in raises a runtime
## Round ID:

Happens on local

## Testmerges:

N/A

## Reproduction:

Clicking on the fading splash after spawning in will throw `Cannot read null.atom_storage` in CanReach().

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

checks to make sure the targets loc isnt null before checking for atom storage

resolves #68365

## Why It's Good For The Game

bug

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
image
The bug is assuming loc will exist
🗞️ 
this fix don't work

**Comment by LemonInTheDark** [comment]
Curious why this check is needed in the first place

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -230,7 +230,7 @@
 			if (!target.loc)
 				continue
 
-			if(target.loc.atom_storage)
+			if(target.loc?.atom_storage)
```
Comment (ninjanomnom) [inline code/_onclick/click.dm:233]: You don't need this one here, loc is already known to not be null due to the if above

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/_onclick/click.dm (hunk 1 above)
Comment (magatsuchi) [inline code/_onclick/click.dm:233]:
```suggestion
			if(target.loc.atom_storage)
```

**Pull request merged**